assets/sass/protocol/includes/_mixins.scss

```diff
@@ -30,69 +28,53 @@
 // Mixins for CSS3 properties that still need prefixes in some browsers.
 @mixin transition($transition...) {
     transition: $transition;
-    transition: $transition;
 }
 
 @mixin transform($transform...) {
     transform: $transform;
-    transform: $transform;
 }
 
 @mixin transform-origin($transform-origin...) {
     transform-origin: $transform-origin;
-    transform-origin: $transform-origin;
 }
 
 @mixin transform-style($transform-style...) {
     transform-style: $transform-style;
-    transform-style: $transform-style;
 }
 
 // Extra mixins for fine-tuned transitions
 @mixin transition-property($transition-property...) {
     transition-property: $transition-property;
-    transition-property: $transition-property;
 }
 
 @mixin transition-duration($transition-duration...) {
     transition-duration: $transition-duration;
-    transition-duration: $transition-duration;
 }
 
 @mixin transition-timing-function($transition-timing-function...) {
     transition-timing-function: $transition-timing-function;
-    transition-timing-function: $transition-timing-function;
 }
 
 @mixin transition-delay($transition-delay...) {
     transition-delay: $transition-delay;
-    transition-delay: $transition-delay;
 }
 
 @mixin animation($parameters...) {
     animation: $parameters;
-    animation: $parameters;
 }
 
 @mixin border-box {
     box-sizing: border-box;
-    box-sizing: border-box;
-    box-sizing: border-box;
 }
 
 @mixin flexbox {
-    display: box;
-    display: box;
-    display: flexbox;
-    display: flex;
     display: flex;
 }
 
 @mixin flex($values: auto) {
     -webkit-box-flex: $values;
     -moz-box-flex: $values;
     flex: $values;
-    flex: $values;
 }
```

Comment:
Issue (possibly blocking?) should these vendor prefixes also be removed? It's not quite clear to me why these are still here if they have been removed elsewhere.